packages/genesys-spark-chart-components/src/components/beta/gux-visualization/gux-visualization.scss

```diff
@@ -17,3 +17,7 @@ text {
 .mark-rule.role-axis-domain line {
   stroke: #2e394c;
 }
+
+.chart-container {
```

Comment:
I know this is annoying and not necessary (as this class is in the shadow dom) but we have a convention of prefixing all classes with `gux-`.